bazel/cc_toolchains/clang_configuration.bzl

```
@@ -76,8 +80,11 @@ def _compute_clang_cpp_include_search_paths(repository_ctx, clang, sysroot):
         # Force the language to be C++.
         "-x",
         "c++",
-        # Read in an empty input file.
-        "/dev/null",
+        # Read in an empty input file. If we are building from
+        # Windows, then we create an empty temp file above
+        # for use here. Clang on Windows does not like it
+        # when you pass a non-existent file.
+        repository_ctx.path("_temp") if repository_ctx.os.name.lower().startswith("windows") else "/dev/null",
```

```suggestion
        input_file,  
```

Following the above.